rules/windows/privilege_escalation_installertakeover.toml

```diff
@@ -14,35 +14,35 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Potential Privilege Escalation via InstallerFileTakeOver"
-note = """## Triage and analysis.
-
-### Investigating Potential Priivilege Escalation via InstallerFileTakeOver
-
-InstallerFileTakeOver is a weaponized EoP PoC to the CVE-2021-41379 vulnerability. Upon successful exploitation,
-an unprivileged user will escalate privileges to SYSTEM/NT AUTHORITY.
-
-This rule detects the default execution of the PoC, which overwrites the `elevation_service.exe` DACL and copies itself
-to the location to escalate privileges. An attacker is able to still take over any file that is not in use (locked), which is outside the scope of this rule.
-
-#### Possible investigation steps:
-
-- Check for the digital signature of the executable.
-- Look for additional processes spawned by the process, command lines, and network communications.
-- Look for additional alerts involving the host and the user.
-
-### False Positive Analysis
-
-- Verify whether the digital signature exists in the executable, and if it is valid.
-
-### Related Rules
-
-- Suspicious DLL Loaded for Persistence or Privilege Escalation - bfeaf89b-a2a7-48a3-817f-e41829dc61ee
-
-### Response and Remediation
-
-- Immediate response steps should be taken to validate, investigate, and potentially contain the activity to prevent
-further post-compromise behavior.
-"""
+note = """## Triage and analysis. 
+ 
+### Investigating Potential Priivilege Escalation via InstallerFileTakeOver 
+ 
+InstallerFileTakeOver is a weaponized EoP PoC to the CVE-2021-41379 vulnerability. Upon successful exploitation, 
+an unprivileged user will escalate privileges to SYSTEM/NT AUTHORITY. 
+ 
+This rule detects the default execution of the PoC, which overwrites the `elevation_service.exe` DACL and copies itself 
+to the location to escalate privileges. An attacker is able to still take over any file that is not in use (locked), which is outside the scope of this rule. 
```

Comment:
what location is meant by "the location"?

Reply:
The original file location, the one that is overwritten